FinniversKit/Sources/Recycling/GridViews/Markets/CompactView/CompactMarketsViewCell.swift

```diff
@@ -147,10 +123,14 @@ extension CompactMarketsViewCell {
 // MARK: - Private extensions
 
 private extension UIColor {
-    class var externalLinkColor: UIColor {
-        dynamicColorIfAvailable(defaultColor: .sardine, darkModeColor: .darkSardine)
-    }
-}
+     class var externalLinkColor: UIColor {
+         return .blueGray600
```

Comment:
does it look nice with dark mode too?

Reply:
Betzy told me to use the same color for both light and dark mode